playbook-website/app/helpers/advanced_table_example_mock_data.rb

```diff
@@ -0,0 +1,280 @@
+# frozen_string_literal: true
+
+TABLE_DATA = [
```

Comment:
I didn't want to put this giant Array inside the doc example, so was looking for the best place for it. It's just for our doc example and does not need to ship with the kit. Is this place ok for this file? @jasperfurniss

Reply:
updating for transparency: discussed with Jason, moving or changing that data file will be covered on a follow up PR, not a blocker on this one.